release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -3279,3 +3279,24 @@ link:https://access.redhat.com/solutions/6936291[{product-title} 4.8.37 containe
 ==== Updating
 
 To update an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-8-39"]
+=== RHBA-2022:1427 - {product-title} 4.8.39 bug fix update
+
+Issued: 2022-04-27
+
+{product-title} release 4.8.39 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1427[RHBA-2022:1427] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1423[RHBA-2022:1423] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6954669[{product-title} 4.8.39 container image list]
+
+[id="ocp-4-8-39-known-issues"]
+==== Known Issue
+
+* Currently, upgrading from 4.8 to 4.9 is not available. {product-title} {product-version} users are advised to upgrade to the next version when it is available in the appropriate release channel. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2068601[*BZ#2068601*]
```

Comment:
{product-title} {product-version} is rendering OpenShift Container Platform Branch Build. I guess it just needs to be {product-title} 4.8 ????

Reply:
also missing closing parenthesis by the BZ#